public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -92,6 +95,79 @@ a#i18n-attribute
   The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file
   to facilitiate contextually-specific translations.
 
+  As an alternative, if you want to translate some content without having to create a new element, you can use a `<ng-container>`
```

Comment:
l 93 
"While all appearance of a message with the _same_ meaning should have the _same_"
should have -> have

" The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file to facilitiate contextually-specific translations."

-> that might not be true depending on the serialization format